README.md

```diff
@@ -52,3 +52,6 @@ active development and is not held to the same standards as `rust-bitcoin` or
 For this reason, it is not recommended to use it in production.
```

Comment:
Can we delete the Stability section now? :)

Reply:
Sure, I can make another PR